src/components/General/SubLabelWrapper/index.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import classnames from "classnames";
+import PropTypes from "prop-types";
+import Label from "./Label";
+
+const SubLabelWrapper = ({
+  subLabelPlacement,
+  children,
+  label,
+  customLabel,
+  name,
+  className,
+}) => {
+  const sub_label_class =
+    subLabelPlacement == "hidden_label"
+      ? "hidden_sub_label screen-reader-text"
+      : "";
+
+  // @TODO respect FORM settings sub label placement
```

Comment:
@Gytjarek This todo doesn't seem to be needed any more.

Reply:
Kind of, it is still needed, as I'm retrieving `subLabelPlacement` from the field settings and I get `INHERIT` when it's not changed, but we don't know what INHERIT means in this case. Therefore, we need to obtain the value from the form settings (TOP or BOTTOM)